### PR TITLE
UI feedback and faster rendering

### PR DESCRIPTION
**Pull request by yangbin**

### CODE REVIEW

**Comment by jieyanhuang** [comment]
:+1:

**Comment by tgohn** [comment]
:+1:

**Comment by yangbin** [comment]
:haircut:

**Comment by jieyanhuang** [comment]
:+1:

File: jx/ui/html/HTMLRenderer.js [hunk 1]
```diff
@@ -101,6 +114,11 @@ HTMLRenderer.prototype.updateElement = function(uid, delta, attr) {
 	if ('fontWeight' in delta)
 		style.fontWeight = delta.fontWeight;
 
+	if ('onclick' in delta)
+		el.onclick = function() {
+			JXML.cast(delta.onclick[0], delta.onclick[1]);
```
Comment (jieyanhuang) [inline jx/ui/html/HTMLRenderer.js]: argh! why never pass in arguments to `JXML.cast`?!

**Pull request merged**